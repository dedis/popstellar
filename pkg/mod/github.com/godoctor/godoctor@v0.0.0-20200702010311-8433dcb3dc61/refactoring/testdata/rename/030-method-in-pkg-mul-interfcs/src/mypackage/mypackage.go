package mypackage 



type Simple interface {

Mymethod() string

} 

type Complex interface {

Mymethod() int

}

type Third interface {

Renamed() string
ThirdOnly()

}

type Mystruct struct {

Myvar string

}

func (mystructvar *Mystruct)Mymethod() string {


return mystructvar.Myvar


}

func (mystructvar *Mystruct) ThirdOnly() {
}
