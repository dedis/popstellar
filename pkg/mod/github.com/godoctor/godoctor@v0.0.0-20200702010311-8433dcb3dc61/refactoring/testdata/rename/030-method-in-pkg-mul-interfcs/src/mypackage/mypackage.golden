package mypackage 



type Simple interface {

Renamed() string

} 

type Complex interface {

Mymethod() int

}

type Third interface {

Renamed() string
ThirdOnly()

}

type Mystruct struct {

Myvar string

}

func (mystructvar *Mystruct)Renamed() string {


return mystructvar.Myvar


}

func (mystructvar *Mystruct) ThirdOnly() {
}
